pmsoService/files/app/repo.py

```
@@ -0,0 +1,81 @@
+        except File.DoesNotExist:
+            return None
+    
+    def delete(self, file_id: UUID) -> None:
+        """Delete File"""
+        File.objects.filter(id=file_id).delete()
+
+
+class FileLinkRepository:
+    """FileLink persistence operations"""
+    
+    def create_link(self, file_id: UUID, model: str, object_id: UUID, meta: dict = None) -> FileLink:
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)